docs/installers.rst

```diff
@@ -583,12 +584,241 @@ Example Linux game:
       game:
         exe: $GAMEDIR/mygame
         args: --some-arg
+        working_dir: $GAMEDIR
 
       files:
       - myfile: http://example.com/mygame.zip
 
       installer:
       - chmodx: $GAMEDIR/mygame
+      system:
+        terminal: true
+        env:
+          SOMEENV: true
+
+Example wine game:
```

Comment:
Is it possible to add anchors to markdown? Would be cool to that a list of those example so we can  quickly switch to them.